js/react/lib/components/contact-form/contact-form.component.tsx

```diff
@@ -0,0 +1,167 @@
+      error: "border-red-500 text-red-600",
+      placeholder: "placeholder-gray-400",
+      button: "bg-orange-500 text-white hover:bg-orange-600",
+      disabled: "bg-gray-300 text-gray-500",
+    },
+    dark: {
+      form: "bg-[#1e1e1e]",
+      bg: "bg-[#2c2c2c] text-white",
+      border: "border-gray-600",
+      focus: "focus:border-orange-400",
+      error: "border-red-500 text-red-500",
+      placeholder: "placeholder-gray-500",
+      button: "bg-orange-500 text-white hover:bg-orange-600",
+      disabled: "bg-gray-700 text-gray-500",
+    },
+  };
+
+  const t = styles[theme];
+
+  return (
+    <form
+      onSubmit={handleSubmit}
+      className={`max-w-xl rounded-2xl p-6 shadow-md ${t.form}`}
+    >
+      <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
+        <div>
+          <input
+            name="name"
+            value={form.name}
+            onChange={handleChange}
+            onBlur={handleBlur}
+            placeholder="Nombre"
+            className={`${base} ${t.bg} ${t.placeholder} ${t.border} ${
+              errors.name ? t.error : touched.name ? t.focus : ""
+            }`}
+          />
+          {errors.name && (
+            <p className="mt-1 text-sm text-red-500">{errors.name}</p>
+          )}
+        </div>
+        <div>
+          <input
+            name="email"
+            value={form.email}
+            onChange={handleChange}
+            onBlur={handleBlur}
+            placeholder="Email"
+            className={`${base} ${t.bg} ${t.placeholder} ${t.border} ${
+              errors.email ? t.error : touched.email ? t.focus : ""
+            }`}
+          />
+          {errors.email && (
+            <p className="mt-1 text-sm text-red-500">{errors.email}</p>
+          )}
+        </div>
+      </div>
+
+      <div className="mt-4">
+        <div className="relative">
+          <Location
+            className={`pointer-events-none absolute top-1/2 left-3 -translate-y-1/2 transform ${
+              theme === "dark" ? "text-gray-400" : "text-gray-500"
+            }`}
+          />
+          <input
+            name="location"
+            value={form.location}
+            onChange={handleChange}
+            onBlur={handleBlur}
+            placeholder="Ubicación"
+            className={`${base} pl-10 ${t.bg} ${t.placeholder} ${t.border} ${
+              errors.location ? t.error : touched.location ? t.focus : ""
+            }`}
+          />
+        </div>
+        {errors.location && (
+          <p className="mt-1 text-sm text-red-500">{errors.location}</p>
+        )}
+      </div>
+
+      <div className="mt-4">
+        <textarea
+          name="message"
+          rows={4}
+          value={form.message}
+          onChange={handleChange}
+          placeholder="Mensaje"
+          className={`${base} ${t.bg} ${t.placeholder} ${t.border} resize-none`}
+        />
+      </div>
+
+      <div className="mt-6 text-right">
+        <button
```

Comment:
We already have the button component created, please use it. It's located at [/js/react/lib/components/button](https://github.com/RustLangES/design-system-components/tree/main/js/react/lib/components/button).